komga-webui/src/functions/book-spreads.ts

```diff
@@ -9,40 +9,36 @@ export function buildSpreads(pages: PageDtoWithUrl[], pageLayout: PagedReaderLay
   if (pageLayout !== PagedReaderLayout.SINGLE_PAGE) {
     const spreads = [] as PageDtoWithUrl[][]
     const pagesClone = cloneDeep(pages)
-    let lastPages = undefined
```

Comment:
Removed lastPages, and the chunk handling it below as the loop part of the algorithm handles getting the last image in place